userspace/libsinsp/examples/test.cpp

```diff
@@ -209,17 +215,24 @@ void open_engine(sinsp& inspector, libsinsp::events::set<ppm_sc_code> events_sc_
 		if (!ppm_sc.empty())
 		{
 			auto events_sc_names = libsinsp::events::sc_set_to_sc_names(ppm_sc);
-			printf("-- Activated ppm sc names in kernel using `sinsp_repair_state_sc_set` enforcement: %s\n", concat_set_in_order(events_sc_names).c_str());
+			printf("-- Activated (%ld) ppm sc names in kernel using `sinsp_repair_state_sc_set` enforcement: %s\n", events_sc_names.size(), concat_set_in_order(events_sc_names).c_str());
 		}
 	}
 
 	if (ppm_sc_modifies_state && !events_sc_codes.empty())
 	{
-		ppm_sc = libsinsp::events::sinsp_state_sc_set().merge(events_sc_codes);
+		if (ppm_sc_state_remove_io_sc)
+		{
+			ppm_sc = libsinsp::events::sinsp_state_sc_set(ppm_sc_state_remove_io_sc).merge(events_sc_codes);
```

Comment:
Actually we don't need this `if` check right? We can eventually just pass `false` and be ok!
So that this `if` just becomes
```
ppm_sc = libsinsp::events::sinsp_state_sc_set(ppm_sc_state_remove_io_sc).merge(events_sc_codes);
```